qemu_v8.mk

```
@@ -259,11 +267,15 @@ $(QEMU_BUILD)/config-host.mak:
 	cd $(QEMU_PATH); ./configure --target-list=aarch64-softmmu --enable-slirp\
 			$(QEMU_CONFIGURE_PARAMS_COMMON)
 
-qemu: $(QEMU_BUILD)/config-host.mak
+qemu: .stamp_qemu
+
+.stamp_qemu: $(QEMU_BUILD)/config-host.mak
 	$(MAKE) -C $(QEMU_PATH)
+	touch $@
 
 qemu-clean:
 	$(MAKE) -C $(QEMU_PATH) distclean
+	rm -f .stamp_qemu
```

Perhaps move `.stamp_qemu` to the QEMU build directory. It is not so good to have a generated file here.
```suggestion
qemu: $(QEMU_BUILD)/.stamp_qemu

$(QEMU_BUILD)/.stamp_qemu: $(QEMU_BUILD)/config-host.mak
	$(MAKE) -C $(QEMU_PATH)
	touch $@

qemu-clean:
	$(MAKE) -C $(QEMU_PATH) distclean
	rm -f $(QEMU_BUILD)/.stamp_qemu
```